### PR TITLE
Resolve #7: Enable basic core modules and add develop modules for con…

### LINKED ISSUE #7: Enable basic core modules
Enable the following core modules during the initial setup: 

- CKEditor
- Field
- Menu UI
- Node
- Path
- Views
- File
- Image
- Link
- Options
- Text

### PR DESCRIPTION
**Pull request by ozin7**
Enable core modules and add develop modules.

### CODE REVIEW

File: draft.info.yml [hunk 1]
```diff
@@ -10,9 +10,22 @@ distribution:
 dependencies:
   # Core
   - drupal:block
-  - drupal:dynamic_page_cache
+  - drupal:ckeditor
   - drupal:config
+  - drupal:dynamic_page_cache
+  - drupal:editor
```
Comment (T2L) [inline draft.info.yml]: Remove as `drupal:ckeditor` depends on it already

File: draft.info.yml [hunk 2]
```diff
@@ -10,9 +10,22 @@ distribution:
...
+  - drupal:field
```
Comment (T2L) [inline draft.info.yml]: Remove as `drupal:ckeditor` depends on it already

File: draft.info.yml [hunk 3]
```diff
@@ -10,9 +10,22 @@ distribution:
...
+  - drupal:link
+  - drupal:media
+  - drupal:menu_link_content
```
Comment (T2L) [inline draft.info.yml]: Remove as `drupal:menu_ui` depends on it already

File: draft.info.yml [hunk 4]
```diff
@@ -10,9 +10,22 @@ distribution:
...
+  - drupal:menu_ui
+  - drupal:node
+  - drupal:options
   - drupal:page_cache
+  - drupal:path
+  - drupal:text
+  - drupal:user
```
Comment (T2L) [inline draft.info.yml]: Remove as `drupal:core` depends on it already

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ## Draft 1.x.x
```
Comment (T2L) [inline CHANGELOG.md]: Move to `Draft 2.x.x` as we are going to do new existing stuff!

File: draft.info.yml [hunk 6]
```diff
@@ -10,9 +10,22 @@ distribution:
...
+  - drupal:views
```
Comment (T2L) [inline draft.info.yml:24]: Add:
- Contextual Links
- Custom block

File: config/install/config_split.config_split.develop.yml [hunk 7]
```diff
@@ -3,8 +3,11 @@ status: true
 dependencies: {  }
 id: develop
 label: Develop
+description: ''
 folder: ../config/develop
-module: {  }
+module:
+  field_ui: 0
```
Comment (T2L) [inline config/install/config_split.config_split.develop.yml:10]: Add:
- Database logging
- Update manager

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
...
-
+- Enable basic core modules and add Views UI and Field UI to the config split settings
```
Comment (T2L) [inline CHANGELOG.md]: Please link issue number, and list all modules

**Review by T2L — CHANGES_REQUESTED** [review]

**Comment by ozin7** [comment]
@T2L I've fixed your comments, please check!

**Comment by ribel** [comment]
@ozin7 please resolve conflicts.

**Pull request merged**